docs/api.md

```diff
@@ -10,7 +10,14 @@ Where `${VMI_NAMESPACE}` and `${VMI_NAME}` are the namespace
 and name of a running VMI.
 
 #### Parameters
-- `duration` - Duration while the token is valid
+- `duration` - Duration while the token is valid. If it is not specified, then the token will expire after 10 minutes.
```

Comment:
```suggestion
- `duration` -  Time in which the token is valid. If it is not specified, then the token will expire after 10 minutes.
```
Feel free to reword my suggestion. But in any case, I find a bit redundant to define "duration" starting with the word "duration".

Reply:
I think that wording "Time in which" is unclear if it is a specific time moment, or a duration.